next.config.js

```diff
@@ -2,6 +2,8 @@
  * @type {import('next').NextConfig}
  */
 
+const withTM = require('next-transpile-modules')(['react-markdown']); // pass the modules you would like to see transpiled
```

Comment:
Resolves issue with `react-markdown` not being set as the next/tsconfig would like it to be.